src/extensions/vectorstores/EmbeddingUtils.py

```diff
@@ -101,6 +101,23 @@
     default_value=OPENAI,
     required=True,
 )
+OPENAI_API_MODEL = PropertyDescriptor(
```

Comment:
I think we should rename this to OPENAI_MODEL, it doesn't really have to do anything with the API